specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/examples/VirtualNetworkGatewaysDisconnectP2sVpnConnections.json

```diff
@@ -0,0 +1,21 @@
+{
+    "parameters": {
+      "subscriptionId": "subid",
+      "resourceGroupName": "vpn-gateway-test",
+      "gatewayName": "vpngateway",
+      "api-version": "2019-11-01",
+      "request": {
+        "vpnConnectionIds": [
+          "vpnconnId1",
+          "vpnconnId2"
+        ]
+      }
+    },
+    "responses": {
+      "200": {
+        "body": ""
```

Comment:
In specs you defined that 200 doesn't return anything, but in example you use empty string. This causes Link checker to throw an error